data/tanitoluwam.json

```
@@ -10,7 +10,7 @@
     },
     {
       "name": "Youtube",
-      "url": "https://youtube.com/tanisjourney7489",
+      "url": "https://www.youtube.com/@tanisjourney7489"
```

Please add the missing comma `,` here. 

```suggestion
      "url": "https://www.youtube.com/@tanisjourney7489",
```